src/frontend/src/pages/WorkPackageTemplateForm/EditWorkPackageTemplate.tsx

```diff
@@ -0,0 +1,41 @@
+import { useEditWorkPackageTemplate, useSingleWorkPackageTemplate } from '../../hooks/work-packages.hooks';
+import WorkPackageTemplateForm from './WorkPackageTemplateForm';
+import { useQuery } from '../../hooks/utils.hooks';
+
+const EditWorkPackageTemplate: React.FC = () => {
+  const query = useQuery();
+
+  const workPackageTemplateId = query.get('workPackageTemplateId');
+
+  const { mutateAsync: editWorkPackageTemplate } = useEditWorkPackageTemplate(workPackageTemplateId!);
+
+  const { data: workPackageTemplate } = useSingleWorkPackageTemplate(workPackageTemplateId!);
+
+  const defaultValues = {
```

Comment:
Import the type for this (which can be renamed WorkPackageTemplateFormInputs or something to declare the type here

You can actually use the spread operator for all of the fields, and will just manually need to override stage and blocked by